OutOfSchool/OutOfSchool.ExternalFileStore.Impl/Gcs/GcpFilesStorageBase.cs

```diff
@@ -101,4 +101,9 @@ protected sealed override async Task DeleteOperationAsync(string fileId, Cancell
     {
         await StorageClient.DeleteObjectAsync(BucketName, fileId, cancellationToken: cancellationToken);
     }
+
+    protected override async Task<bool> ExistsOperationAsync(string imageId, CancellationToken cancellationToken = default)
+    {
+        return (await StorageClient.GetObjectAsync(BucketName, imageId, cancellationToken: cancellationToken) is null);
```

Comment:
do i understand this correctly:
if client returns something that is `not null` - we get `false`, if it returns something that is `null` - we get `true`? Shouldn't it be other way around?

p.s. Maybe there's something how the google client works that i don't know :)